debug/org.eclipse.cdt.debug.core/src/org/eclipse/cdt/debug/core/launch/CoreBuildLaunchBarTracker.java

```diff
@@ -109,7 +110,23 @@ private void setActiveBuildConfig(ILaunchMode mode, ILaunchDescriptor descriptor
 		lastDescriptor = descriptor;
 		lastTarget = target;
 
-		// Pick build config based on toolchain for target
+		/*
+		 * Core build projects do not work with the concept of active build
+		 * configurations, like managed build projects. Instead they rely
+		 * on the launch mode set in the launchBar. A core build launch
+		 * configuration looks at the launchBar mode to pick the build
+		 * configuration. Core build projects still have an active build
+		 * configuration, but it is hidden for the user and ignored for
+		 * launching.
```

Comment:
* Core build projects do not work with the concept of active build
		 * configurations, like managed build projects. Instead they rely
		 * on the launch mode and launch target set in the launchBar. A core build launch
		 * configuration looks at the launchBar launch mode and then the set of toolchains
		 * associated with the launch target to pick the build configuration.
		 * Core build projects still have an active build
		 * configuration, but it is hidden for the user and ignored for
		 * launching.

Reply:
I converted this comment to a suggested code change by using this button in the review:

![image](https://user-images.githubusercontent.com/679236/234960148-aebfc1c2-4389-4bfe-874b-cdec96011e67.png)

This is one of the few features that makes github reviewing better than gerrit IMHO.